src/core/commands.rs

```diff
@@ -408,10 +408,7 @@ async fn modify_message(params: CallBackParams) -> CallbackReturn {
     )
   } else {
     (
-      (
-        params.message.channel_id.0 as u64,
-        parse::DiscordIds::Channel,
-      ),
+      (params.message.channel_id.0, parse::DiscordIds::Channel),
```

Comment:
Do not push modification on files you don't edit

Reply:
There was a clippy warning due to the cast
```
casting to the same type is unnecessary (`u64` -> `u64`)
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_cast
`#[warn(clippy::unnecessary_cast)]` implied by `#[warn(clippy::all)]`
```

Reply:
oh ! didn't see that :+1: